packages/freshchat-api/README.md

```diff
@@ -32,6 +40,23 @@ freshchat.conversationStatusUpdate('<conversation-id>', 'resolved');
 freshchat.conversationStatusUpdate('<conversation-id>', 'new');
 ```
 
+### Generate chat Transcript
+```javascript
 
-
-
+var app_url = 'https://domain.freshchat.com';
+var app_alias = '<App ID>'; //Available from Settings--> Mobile SDKs
+var conversationID = '<Freshchat Conversation UUID>';
```

Comment:
We generally use camelCase in all JS code. 
Please use `appUrl`, `appAlias`, `conversationId` for the sake of convention.